test/config.test.js

```diff
@@ -204,27 +230,42 @@ describe('javascript config', () => {
         expect(config.js.lintPaths).toEqual(lintPaths);
     });
 
-    it('dist file should be set', () => {
-        expect(config.js.files.main.distFile).toBe('script.js');
+    it('`usePackageVersion` should exist', () => {
+        expect(config.js.usePackageVersion).toBeDefined();
     });
 
-    it('dist file can be updated', () => {
+    it('`usePackageVersion` should be set to `false` by default', () => {
+        expect(config.js.usePackageVersion).toBe(false);
+    });
+
+    it('usePackageVersion can be updated', () => {
         // Arrange
-        const distFile = 'app.js';
+        const usePackageVersion = true;
 
         // Act
-        config.update({ js: { files: { main: { distFile } } } });
+        config.update({ js: { usePackageVersion } });
 
         // Assert
-        expect(config.js.files.main.distFile).toBe(distFile);
+        expect(config.js.usePackageVersion).toBe(usePackageVersion);
     });
 
-    it('`usePackageVersion` should exist', () => {
-        expect(config.js.usePackageVersion).toBeDefined();
+    it('`stripDebug` should exist', () => {
+        expect(config.js.stripDebug).toBeDefined();
     });
 
-    it('`usePackageVersion` should be set to `false` by default', () => {
-        expect(config.js.usePackageVersion).toBe(false);
+    it('`stripDebug` should be set to `true` by default', () => {
+        expect(config.js.stripDebug).toBe(true);
+    });
+
+    it('strip debug can be updated', () => {
```

Comment:
`stripDebug`.

Reply:
@kevinrodrigues Should I change the ones that say 'is production' as well then?

Reply:
I think so. There's a mixture of descriptions in this file, usually I add the 'thing' I'm talking about or testing as "`methodName`" `` (with backticks) so it's clear what we're actually referring too in the test itself, I've also found it helps to find that reference quicker when a test fails. :)

Reply:
Let's keep things consistent with these names? All of the existing tests use full, space separated words. I don't mind which one we go with as long as they are consistent :sierra_leone:

Reply:
I've chosen to use the variable names as they are found in the config, but with backticks to make it clearer.